angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -170,10 +170,19 @@
                 <nimble-list-option *ngFor="let item of comboboxItems" [ngValue]="item">{{ item ? item.first : '' }}</nimble-list-option>
             </nimble-combobox>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Rich Text Editor</div>
+            <div class="rich-text-editor-container">
+                <nimble-rich-text-editor placeholder="Add your text here..." (input)="onEditorInput()" #editor>
+                    <nimble-button slot="footer-actions" (click)="setMarkdown()">Set</nimble-button>
+                    <nimble-button [disabled]="isEditorButtonDisabled" slot="footer-actions" (click)="getMarkdown()">Get & View</nimble-button>
```

Comment:
nit: can we consider removing all the event functionalities (similar to the other examples)

Reply:
Removed the event and method functionalities as we discussed.